src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/shared/resources/isLoading/stateless/statelessIsLoadingSagas.ts

```diff
@@ -0,0 +1,20 @@
+import { SagaIterator } from 'redux-saga';
+import { take, all, put } from 'redux-saga/effects';
+import FormDataActions from 'src/features/form/data/formDataActions';
+import { FETCH_SERVICE_CONFIG_FULFILLED } from 'src/features/form/dynamics/formDynamicsActionTypes';
+import { FormLayoutActions } from 'src/features/form/layout/formLayoutSlice';
+import { FETCH_RULE_MODEL_FULFILLED } from 'src/features/form/rules/rulesActionTypes';
+import { startInitialStatelessQueue } from '../../queue/queueSlice';
+import { finishStatelessIsLoading } from '../isLoadingSlice';
+
+export function* watcherFinishStatelessIsLoadingSaga(): SagaIterator {
+  yield take(startInitialStatelessQueue);
+  yield all([
+    take(FormDataActions.fetchFormDataFulfilled),
```

Comment:
Should we wait for data model here as well?

Reply:
Jupp, great catch. Fixed.